code/datums/abilities/werewolf.dm

```diff
@@ -94,14 +94,12 @@
 				src.bioHolder.AddEffect("regenerator")
 				boutput(src, "<span style=\"color:red\">You will now heal over time!</span>")
 
-			if (M.handcuffed)
-				if (M.handcuffed.werewolf_cant_rip())
+			if (M.hasStatus("handcuffed"))
+				if (M.handcuffs.werewolf_cant_rip())
```

Comment:
Are we okay assuming that if you have the handcuffed status then `M.handcuffed` is a set of handcuffs?

Reply:
It'd be `M.handcuffs`, but yes. You can only get the status by setting the var, and vice-versa. Short of admin var editing anyways.